### PR TITLE
Magiclysm update

### PR DESCRIPTION
**Pull request by Kenan2000**
#### Summary
SUMMARY: Mods "Update Magiclysm to include all the newest content in it"

#### Purpose of change
Updating Magiclysm mod to include all the newest changes to it and new content

#### Describe the solution
Update Magiclysm mod

#### Describe alternatives you've considered
Do not update Magiclysm mod

#### Testing
Tested professions list, debug item spawn list, debug monster spawn list, thoroughly analyzed the code and changes, working nicely

#### Additional context
None

### CODE REVIEW

File: data/mods/Magiclysm/Spells/attunements/Artificer.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
+[
+  {
+    "id": "artificer_toolkit",
+    "type": "GENERIC",
+    "name": { "str": "artificer's toolkit" },
+    "description": "This is a magical device that can take the shape of many tools.  When not in use it looks like a simple screwdriver.",
+    "looks_like": "screwdriver",
+    "weight": "170 g",
+    "volume": "40 ml",
+    "longest_side": "25 cm",
```
Comment (Coolthulhu) [inline data/mods/Magiclysm/Spells/attunements/Artificer.json:10]: Not supported

File: data/mods/Magiclysm/items/ethereal_items.json [hunk 2]
```diff
@@ -171,6 +171,7 @@
     "max_damage": 0,
     "effect": "spawn_item",
     "effect_str": "stormglove",
+    "shape": "blast",
```
Comment (Coolthulhu) [inline data/mods/Magiclysm/items/ethereal_items.json:174]: Not supported

File: data/mods/Magiclysm/items/ethereal_items.json [hunk 3]
```diff
@@ -463,5 +496,205 @@
...
+        {
+          "has": "WORN",
+          "condition": "ALWAYS",
+          "values": [
+            { "value": "STRENGTH", "add": 3 },
+            { "value": "DEXTERITY", "add": 3 },
+            { "value": "INTELLIGENCE", "add": -3 },
+            { "value": "PERCEPTION", "add": -3 }
+          ],
+          "mutations": [ "LIGHTFUR", "CLAWS", "ADRENALINE" ]
```
Comment (Coolthulhu) [inline data/mods/Magiclysm/items/ethereal_items.json:534]: Not supported

File: data/mods/Magiclysm/mutations/dragon_mutation_category.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "type": "mutation_category",
+    "id": "DRAGON_BLACK",
+    "name": "Black Dragon",
+    "wip": true,
```
Comment (Coolthulhu) [inline data/mods/Magiclysm/mutations/dragon_mutation_category.json:6]: Comments should start with "//"

File: data/mods/Magiclysm/traits/attunements.json [hunk 5]
```diff
@@ -0,0 +1,1132 @@
...
+    "type": "mutation",
+    "name": { "str": "Artificer" },
+    "points": 8,
+    "starting_trait": false,
+    "purifiable": false,
+    "valid": false,
+    "description": "With the mechanically-minded Technomancer and the sheer power of lightning from a Stormshaper, the Artificer has a unique confluence of skills and spells to be perfect for invention.  While modern inventions and machines have won out in a modern society, after a cataclysm one has a chance to go back to roots of things.  To make weird gadgets that are just as likely to explode as explode the thing you're pointing at.",
+    "prereqs": [ "STORMSHAPER", "TECHNOMANCER" ],
+    "spells_learned": [ [ "artificer_toolkit", 5 ] ],
+    "crafting_speed_multiplier": 1.2,
```
Comment (Coolthulhu) [inline data/mods/Magiclysm/traits/attunements.json:13]: Not supported

File: data/mods/Magiclysm/traits/attunements.json [hunk 6]
```diff
@@ -0,0 +1,1132 @@
...
+    "type": "mutation",
+    "name": { "str": "Biotek" },
+    "points": 8,
+    "starting_trait": false,
+    "purifiable": false,
+    "valid": false,
+    "description": "The Biotek is quite simply the marriage of technology and flesh.  It was not even discovered as an attunement until recent times due to the great advancements in technology in the last 50 years.  The Biotek is unique as it is the only known attunement that warps the Aethereum in such a way as to allow cybernetics to coincide with magical abilities, to an extent.",
+    "prereqs": [ "BIOMANCER", "TECHNOMANCER" ],
+    "spells_learned": [ [ "biotek_healing", 5 ] ],
+    "bionic_mana_penalty": 0.5,
```
Comment (Coolthulhu) [inline data/mods/Magiclysm/traits/attunements.json:87]: Not supported

File: data/mods/Magiclysm/traits/attunements.json [hunk 7]
```diff
@@ -0,0 +1,1132 @@
...
+    "type": "mutation",
+    "name": { "str": "Shapeshifter" },
+    "points": 8,
+    "starting_trait": false,
+    "purifiable": false,
+    "valid": false,
+    "description": "The shapeshifter is the Biomancer refined - able to use a flicker of mana to change shape at the drop of a hat to make yourself more inviting or shape others into desired shapes.",
+    "prereqs": [ "BIOMANCER", "MAGUS" ],
+    "//": "Purposefully omits torso, head, etc.",
+    "encumbrance_multiplier_always": { "ARM_L": 0.4, "ARM_R": 0.4, "HAND_L": 0.4, "HAND_R": 0.4, "LEG_L": 0.4, "LEG_R": 0.4, "MOUTH": 0.4 },
```
Comment (Coolthulhu) [inline data/mods/Magiclysm/traits/attunements.json:790]: Not supported

File: data/mods/Magiclysm/Spells/attunements/Biotek.json [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+    "min_damage": -5,
+    "max_damage": -40,
+    "damage_increment": -1.0,
+    "max_level": 35,
+    "effect": "target_attack",
+    "base_casting_time": 500,
+    "base_energy_cost": 800,
+    "energy_increment": -20,
+    "final_energy_cost": 100,
+    "components": "jury_rig",
```
Comment (Coolthulhu) [inline data/mods/Magiclysm/Spells/attunements/Biotek.json:22]: Not supported

File: data/mods/Magiclysm/worldgen/house_technomancer.json [hunk 9]
```diff
@@ -0,0 +1,270 @@
...
+    "weight": 250,
+    "object": {
+      "fill_ter": "t_floor",
+      "rows": [
+        "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~",
+        "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~",
+        "~~~~~~~~#::::#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~",
+        "~~~~~~~##THHy#####~~~~~~~~~~~~~~~~###o####o###~~",
+        "~~~~~~^#Y    R|> #~~~~~~~~~~~~~~~~#2O5746 yxx#~~",
+        "~~~~~~~:I    ∞| y:~~~~~~~~~~~~#&&&#1,,,JA    #~~",
```
Comment (Coolthulhu) [inline data/mods/Magiclysm/worldgen/house_technomancer.json:116]: ```Terminated: line 9:62: format: rows: row 6 column 14: '∞' has no terrain, furniture, or other definition```

File: data/mods/Magiclysm/traits/attunements.json (hunk 6 above)
Comment (Kenan2000) [reply data/mods/Magiclysm/traits/attunements.json:87]: I'll slowly get to all of these in some while...if you don't mind :-)

File: data/mods/Magiclysm/items/ethereal_items.json (hunk 3 above)
Comment (chaosvolt) [reply data/mods/Magiclysm/items/ethereal_items.json:534]: Porting over the code needed for this will be on my todo list.

**Comment by Coolthulhu** [comment]
Untouched for few months. I assume it's dead.

**Pull request closed without merging**